atd-vze/src/Components/GridExportData.js

```diff
@@ -159,6 +183,7 @@ const GridExportData = ({ query, columnsToExport, totalRecords }) => {
               className=""
               data={formatExportData(data[query.table])}
               filename={query.table + moment(Date.now()).format()}
+              // separator={"|"}
```

Comment:
Did changing the separator work out when you tested it?

Reply:
Thanks for catching this, meant to remove it. It does what you'd expect but requires a bit more configuration on the import side for whatever software you're using. Was pretty easy to figure out with Numbers but it doesn't solve the problem of returns.

Slacking you an example export with pipe delimiters.